streaming/src/event_service.cc

```diff
@@ -0,0 +1,196 @@
+#include <unordered_set>
+
+#include "event_service.h"
+namespace ray {
+namespace streaming {
+
+EventQueue::~EventQueue() {
+  is_started_ = false;
+  no_full_cv_.notify_all();
+  no_empty_cv_.notify_all();
+};
+
+void EventQueue::Start() { is_started_ = true; }
```

Comment:
Can we remove `EventQueue::Start/Stop`. It's a little strange for a blocking queue. Maybe we reabstract this.

Reply:
> Can we remove `EventQueue::Start/Stop`. It's a little strange for a blocking queue. Maybe we reabstract this.

`Stop` means EventQueue is prohibited to write.

Reply:
I rename them to `Unfreeze` and `Freeze`.